src/components/Lightbox.js

```diff
@@ -44,7 +44,7 @@ const propTypes = {
     activeIndex: PropTypes.number,
 
     /** Additional styles to add to the component */
-    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+    style: PropTypes.oneOfType([PropTypes.bool, PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
```

Comment:
```suggestion
    style: stylePropTypes,
```

In `index.native.tsx` add this comment:

```js
            // @ts-expect-error TODO: Remove this once Lightbox (https://github.com/Expensify/App/pull/33081) is migrated to TypeScript.
            style={style}

```